Tests/ActionTests/ActionTests.swift

```diff
@@ -91,6 +105,10 @@ class ActionTests: QuickSpec {
                             next(20, false),
                         ])
                     }
+
+                    it("executes twice") {
+                        XCTAssertEqual(executionObservables.events.count, 2)
```

Comment:
Yes, we can use Nimble matchers here. I used XCTAssertEqual in other places for matching `Recorded<Event<T>>`, which is provided in [RxTests](https://github.com/ReactiveX/RxSwift/blob/master/RxTest/XCTest%2BRx.swift), but other than that, Nimble matchers might be better choice.

I'll replace the matchers that can be replaced by Nimble matchers.